modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -79,24 +79,30 @@ Suppose that you use the following snippet in your `ClusterLogForwarder` CR YAML
 
 [source,yaml]
 ----
-outputDefaults:
+outputDefaults: <1>
  elasticsearch:
-    structuredTypeKey: openshift.labels.myLabel <1>
+    structuredTypeKey: kubernetes.labels.managed <2>
     structuredTypeName: nologformat
+outputs:
+  - name: elasticsearch-secure
+    elasticsearch:
+      structuredTypeKey: openshift.labels.unmanaged
+      structuredTypeName: nologformat
 pipelines:
- - name: application-logs
-   inputRefs:
-   - application
-   - audit
-   outputRefs:
-   - elasticsearch-secure
-   - default
-   parse: json
-   labels:
-     myLabel: myValue <2>
+  - name: application-logs
+    inputRefs:
+    - application
+    - audit
+    outputRefs:
+    - elasticsearch-secure
+    - default
+    parse: json
+    labels:
+      myLabel: myValue <3>
 ----
-<1> Uses the value of the key-value pair that is formed by the OpenShift `myLabel` label.
-<2> The `myLabel` element gives its string value, `myValue`, to the structured log record.
+<1> `outputDefaults` applies only to the default Elasticsearch logstore and not to the other defined Elasticsearch outputs.
+<2> Uses the value of the key-value pair that is formed by the OpenShift `myLabel` label.
```

Comment:
the ... 'managed'... label

Reply:
Made updates as discussed in slack thread: https://redhat-internal.slack.com/archives/GGUR75P60/p1718034341293289